src/test_l1_tvSettings.c

```diff
@@ -366,12 +354,13 @@ void test_l1_tvSettings_positive_RegisterVideoFormatChangeCB (void)
  * **Test Procedure:**@n
  * | Variation / Step | Description | Test Data | Expected Result | Notes |
  * | :-------: | ------------- | --------- | --------------- | ----- |
- * | 01 | call RegisterVideoFormatChangeCB() -  Register the callback function even before TvInit() | tvVideoFormatCallbackData& | void | tvERROR_INVALID_STATE | Should Pass |
+ * | 01 | call RegisterVideoFormatChangeCB() -  Register the callback function even before TvInit() | tvVideoFormatCallbackData * | void | tvERROR_INVALID_STATE | Should Pass |
```

Comment:
Remove void, already test data is added as tvVideoFormatCallbackData *

Reply:
Done